addons/better-featured-project/userstyle.css

```diff
@@ -142,3 +142,8 @@
 .sa-user-id {
   color: white;
 }
+.location {
+  word-break: break-all;
+  width: 660px;
```

Comment:
This width would need to change depending on the width of the `Change featured project` button (doing this dynamically with JavaScript would be overkill, let's not do that). Some languages might use a longer string for this. So I suggest using a smaller widththat will work for the vast majority of languages, maybe 600px.